doc/install_sentry.rst

```diff
@@ -0,0 +1,125 @@
+Installing Docker Sentry
+========================
+
+Install docker
+--------------
+
+Install docker-ce:
+~~~~~~~~~~~~~~~~~~
+
+Follow the steps of this link under the **Install Docker CE** section
```

Comment:
You can add an anchor here like this:
```suggestion
Follow the steps of this link under the `Install Docker CE`_ section
.. _`Install Docker CE`: https://docs.docker.com/install/linux/docker-ce/ubuntu/#install-docker-ce
```

And the second part with the link is usually shown at the bottom of the rst file, so you can remove the link.